notebooks/photutils/03_aperture_photometry/03_aperture_photometry.ipynb

```diff
@@ -0,0 +1,970 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With `photutils`, users can create apertures with the following shapes:\n",
+    "\n",
+    "<a><img src=\"apertures.png\" width=700 alt=\"Examples of circular, elliptical, and rectangular apertures and annuli.\"></a>\n",
+    "\n",
+    "Each of these can be defined either in pixel coordinates or in celestial coordinates (using a WCS transformation).\n",
+    "\n",
+    "It is also possible for users to create custom aperture shapes.\n",
+    "\n",
+    "Any aperture object is created by defining its position and size (and, if applicable, orientation). Let's use the `find_peaks` method that we learned about in a [previous notebook](../02_source_detection/02_source_detection.ipynb) to get the positions of sources in our data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from photutils import find_peaks\n",
+    "from photutils.centroids import centroid_2dg"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Calculate statistics\n",
+    "mean, median, std = sigma_clipped_stats(xdf_image.data, sigma=3.0, maxiters=5, mask=xdf_image.mask)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sources_findpeaks = find_peaks(xdf_image.data, mask=xdf_image.mask, \n",
+    "                               threshold=20.*std, box_size=30, \n",
+    "                               centroid_func=centroid_2dg)     \n",
+    "# Display the table\n",
+    "sources_findpeaks"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And let's plot the centroids of each of these sources:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set up the figure with subplots\n",
+    "fig, ax1 = plt.subplots(1, 1, figsize=(8, 8))\n",
+    "\n",
+    "# Plot the data\n",
+    "fitsplot = ax1.imshow(np.ma.masked_where(xdf_image.mask, xdf_image_clipped), norm=norm_image)\n",
+    "ax1.scatter(sources_findpeaks['x_centroid'], sources_findpeaks['y_centroid'], s=10, marker='.', \n",
+    "            lw=1, alpha=0.7, color='r')#facecolor='None', edgecolor='r')\n",
+    "\n",
+    "# Define the colorbar\n",
+    "cbar = plt.colorbar(fitsplot, fraction=0.046, pad=0.04, ticks=LogLocator(subs=range(10)))\n",
+    "labels = ['$10^{-4}$'] + [''] * 8 + ['$10^{-3}$'] + [''] * 8 + ['$10^{-2}$']\n",
+    "cbar.ax.set_yticklabels(labels)\n",
+    "\n",
+    "# Define labels\n",
+    "cbar.set_label(r'Flux Count Rate ({})'.format(xdf_image.unit.to_string('latex')), \n",
+    "               rotation=270, labelpad=30)\n",
+    "ax1.set_xlabel('X (pixels)')\n",
+    "ax1.set_ylabel('Y (pixels)')\n",
+    "ax1.set_title('find_peaks Sources')"
```

Comment:
Just a generic message about my latex issues which will presumably be an issue throughout this notebook too... Not sure what the fix is, but I did see you removed a `usetex` line from the `.mplstyle`... Maybe if that's added back in (and whatever other issues causing its removal can be fixed) that'll mean this can consistently be changed?